imagedephi/redact.py

```diff
@@ -131,8 +131,24 @@ def _get_output_path(file_path: Path, output_dir: Path) -> Path:
     return output_dir / f"REDACTED_{file_path.name}"
 
 
+def _save_redacted_tiff(tiff_info: TiffInfo, output_path: Path, input_path: Path, overwrite: bool):
+    if overwrite or not output_path.exists():
+        if output_path.exists():
+            click.echo(f"Found existing redaction for {input_path.name}. Overwriting...")
+        tifftools.write_tiff(tiff_info, output_path, allowExisting=True)
+    else:
+        click.echo(
+            f"Could not redact {input_path.name}, existing redacted file in output directory. "
+            "Use the --overwrite-existing-output flag to overwrite previously redacted files."
+        )
```

Comment:
```suggestion
    if output_path.exists():
        if overwrite:
            click.echo(f"Found existing redaction for {input_path.name}. Overwriting...")
        else:
           click.echo(
               f"Could not redact {input_path.name}, existing redacted file in output directory. "
               "Use the --overwrite-existing-output flag to overwrite previously redacted files."
           )
           return
    tifftools.write_tiff(tiff_info, output_path, allowExisting=True)
```

I think this logic structure is a bit easier to follow, but it's up to you.